demo/notebook/cached_spleen_segmentation_3d.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "5141c4a3-4305-41f7-8e58-cbdcd5d28201",
+   "metadata": {},
+   "source": [
+    "# Spleen 3D segmentation with MONAI\n",
+    "\n",
+    "This tutorial uses cached data generated by the Spleen 3D Segmentation tutorial from MONAI to demonstrate how to use the TensorboardPlugin3D.\n",
+    "\n",
+    "The original Spleen dataset can be downloaded from http://medicaldecathlon.com/.\n",
+    "\n",
+    "![spleen](http://medicaldecathlon.com/img/spleen0.png)\n",
+    "\n",
+    "Target: Spleen  \n",
+    "Modality: CT  \n",
+    "Size: 61 3D volumes (41 Training + 20 Testing)  \n",
+    "Source: Memorial Sloan Kettering Cancer Center  \n",
+    "Challenge: Large ranging foreground size\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/KitwareMedical/tensorboard-plugin-3d/blob/main/demo/notebook/cached_spleen_segmentation_3d.ipynb)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e962c9f6-135f-4034-a008-f2b54f70db45",
+   "metadata": {},
+   "source": [
+    "## Setup environment"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "4be79f1e-e133-40d8-8c60-a534af4d43b8",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mWARNING: You are using pip version 21.3.1; however, version 22.0.3 is available.\n",
+      "You should consider upgrading via the '/home/local/KHQ/brianna.major/.virtualenvs/tensorboard-plugin/bin/python -m pip install --upgrade pip' command.\u001b[0m\n"
```

Comment:
local path

Reply:
This was part of the output that needed to be removed. This is now cleaned up.